src/core/issues-notes.js

```diff
@@ -23,6 +23,16 @@ export function run(conf, doc, cb) {
         "<div><h2>" + conf.l10n.issue_summary + "</h2><ul></ul></div>"
       ),
       $issueList = $issueSummary.find("ul");
+    if (!(Object.keys(ghIssues).length === 0 && ghIssues.constructor === Object)) {
+      $ins = $ins.filter(function (index, item) {
+        var dataNum = item.getAttribute("data-number");
+        if (ghIssues[dataNum] === undefined) {
+          item.remove();
```

Comment:
Filtering should have no side effects (i.e., don't `remove()` things)... return the things you are going to remove, then call `.forEach()` to remove them.

Reply:
ok

Reply:
so shall I keep this or remove? 
```javascript
Object.keys(ghIssues).length === 0
```

Reply:
Probably not

Reply:
Check is probably not needed, that is

Reply:
Ok marcos I'll change this once I find some time. Currently focusing on my proposal :sweat_smile:

Reply:
![image](https://user-images.githubusercontent.com/30361728/36377184-2a3313e4-159c-11e8-9d5b-39a305fe0e6c.png)
What do I do in such a case?